test/functional/interface_rpc.py

```diff
@@ -45,7 +45,7 @@ def test_batch_request(self):
             # work fine.
             {"method": "invalidmethod", "id": 2},
             # Another call that should succeed.
-            {"method": "getbestblockhash", "id": 3},
+            {"method": "getblockhash", "id": 3, "params": [self.nodes[0].getblockcount()]},
```

Comment:
Concept ACK on testing a call with parameters.
Though, you're calling into the RPC while building the query, which is a bit funny (but not a problem in practice as there is a well-defiend execution order).

Reply:
doesn't reflect practical usage sure, but was a quick way to demonstrate how it's done, and exercise it :)

I was looking for an example and tests are the first place I look!

Reply:
Could just use `"params": [0]`.

Reply:
fair enough that's easy to understand, done